chromium/rules.js

```diff
@@ -243,20 +246,13 @@ RuleSets.prototype = {
     var rules = ruletag["rule"];
     for (let rule of rules) {
       if (rule["from"] != null && rule["to"] != null) {
-        rule_set.rules.push(new Rule(rule["from"], rule["to"]));
+        rule_set.rules.push(getRule(rule["from"], rule["to"]));
       }
     }
 
     var exclusions = ruletag["exclusion"];
     if (exclusions != null) {
-      for (let exclusion of exclusions) {
-        if (exclusion != null) {
-          if (!rule_set.exclusions) {
-            rule_set.exclusions = [];
-          }
-          rule_set.exclusions.push(new Exclusion(exclusion));
-        }
-      }
+      rule_set.exclusions = new RegExp(exclusions)
```

Comment:
`ruletag["exclusion"]` is an array of `exclusion` if it is not empty.

Reply:
Not anymore (in merged JSON) - it's just a single regexp string.

Reply:
Sorry, I've overlooked. It looks good for me now. thanks for this PR!!